tests/py3-expected-failures.txt

```diff
@@ -1 +0,0 @@
-/h/tests/h/schemas/annotation_test.py:197:
```

Comment:
Only one line is being removed here even though there were two separate fixes. This is because the `form_test.py` failure was missing due to the way the error list was extracted from the test output.